Source/JavaScriptCore/assembler/MacroAssemblerX86_64.h

```diff
@@ -2932,7 +2932,7 @@ class MacroAssemblerX86_64 : public MacroAssemblerX86Common {
     {
         RELEASE_ASSERT(supportsAVX());
         RELEASE_ASSERT(simdInfo.lane == SIMDLane::v128);
-        m_assembler.vandnps_rrr(right, left, dest);
+        m_assembler.vandnps_rrr(left, right, dest); // left and right are intentionally reversed to achieve the desired semantics
```

Comment:
Can you describe what the semantics difference is in a comment?

Reply:
Added a comment above in the latest push.